src/Message.js

```diff
@@ -82,7 +82,7 @@ export default class Message extends React.Component {
       <View>
         {this.renderDay()}
         <View style={[styles[this.props.position].container, {
-          marginBottom: this.isSameUser(this.props.currentMessage, this.props.nextMessage) ? 2 : 10,
+          marginBottom: this.isSameUser(this.props.currentMessage, this.props.nextMessage) ? this.props.marginSameUser : this.props.marginDifferentUser,
```

Comment:
@jayshah123 Don't worry.

You only need to create a new branch in your fork, make the commit on it and create a new PR in the same way you created the previous one.

My recommendation is that this new branch will be created from `master` (in this case).

The Github documentation is better explained that my comment. Here you have some links. I hope it'll be useful for you.

https://guides.github.com/introduction/flow/
https://help.github.com/articles/creating-a-pull-request/